charts/hono/Chart.yaml

```diff
@@ -15,9 +15,9 @@ name: hono
 description: |
   Eclipse Hono™ provides remote service interfaces for connecting large numbers of IoT devices to a back end and
   interacting with them in a uniform way regardless of the device communication protocol.
-version: 2.3.0
+version: 2.3.1
 # Version of Hono being deployed by the chart
-appVersion: 2.3.0
+appVersion: 2.3.1
```

Comment:
Only `version` (i.e. chart version) needs to be changed here.
`appVersion` refers to the Hono release version to be used - it needs to stay unchanged here.

Reply:
Thank you, its changed to proper values now.